modules/admin_manual/pages/maintenance/backup_and_restore/backup.adoc

```diff
@@ -1,69 +1,94 @@
+Depending how the ownCloud instance has been installed, you may need slightly different steps to do a backup. You may also use diffrent methods as the data directory can be huge. This document is intended as a guideline, but the way you implement it depends on your setup.
 
-When backing up your ownCloud server, you need to copy the following things in your ownCloud folder (by default located under `/var/www/`):
+In any case you need the following components to be backed up:
 
-. the `config/` directory containing the main configuration file `config.php` plus sample configurations and possibly custom configurations,
-. the `data/` directory containing web-server-related files, `owncloud.log` and possibly encryption keys,
-. a fresh ownCloud database dump,
-. and if applicable custom theme files which are usually located in the `apps` or `apps-external` directory (see xref:developer_manual:core/theming.adoc[Theming ownCloud]).
+.  The `config/` directory.
+.  The `data/` directory. +
+Note that the `data/` directory may not only contain user files, but also keys for encryption.
+.  The `apps/` directory. +
+Note that this is only necessary if you are not using the `apps-external/` directory and have added own apps or themes.
+.  The `apps-external/` directory. +
+Note that this is only necessary if available.
+.  The ownCloud database.
+.  The custom theme files, if you had any. See xref:developer_manual:core/theming.adoc[Theming ownCloud]. +
+Note that theme files are usually located in either the `apps/` or `apps-external/` directory.
 
-If you install your ownCloud server from our
-https://download.owncloud.org/download/repositories/stable/owncloud/[Open
-Build Service] packages (or from distro packages, which we do not
-recommend), *do not backup your ownCloud server files*, which are the
-other files in your `owncloud/` directory such as `core/`, `3rdparty/`,
-`apps/`, `lib/`, and all the rest of the ownCloud files. If you restore
-these files from backup they may not be in sync with the current package
-versions and will fail the code integrity check. This may also cause
-other errors.
+IMPORTANT: In case you have customized user home direcories or a custom location for encryption keys, you have to manually take care backing them up and restoring them to the same location.
 
-If you install ownCloud from the source tarballs, this will not be an
-issue, and you can safely backup your entire ownCloud installation, with
-the exception of your ownCloud database. Databases cannot be copied, instead
-you must use the database tools to make a correct database dump.
+== Prerequisites
 
-To restore your ownCloud installation from backup, see xref:maintenance/backup_and_restore/restore.adoc[Restoring ownCloud].
+To secure your backup process, stop the web server until backup has finished to prevent user access:
 
-== Backing up the config/ and data/ Directories
+[source,console,subs="attributes+"]
+----
+sudo service apache2 stop
```

Comment:
I guess this is a guide for people that don't already have a big IT infrastructure and backup plans...
Stopping Apache is going to stop all web services that are being served by the system. That might be only ownCloud, but it might be a lot of unrelated stuff.

Reply:
You are totally right, therefore I have added at the top:

> This document is intended as a guideline, but the way you implement it depends on your setup

Open for suggestions to make this more eye catching

Reply:
fixed, relaxed the apache part